packages/nimble-components/src/rich-text/editor/tests/rich-text-editor.spec.ts

```diff
@@ -1052,6 +1052,395 @@ describe('RichTextEditor', () => {
                     );
                 }
             });
+
+            describe('pasting various valid(https/http) links should render as absolute links', () => {
+                const differentProtocolLinks = [
```

Comment:
nit: I don't think you need to test every permutation. It would probably be sufficient to have a couple cases for absolute vs. hyperlink, then maybe, for example, one testing an uppercase https, and one testing a mixed case http.

This is mostly a suggestion for the future. If you want to leave all these test cases, you can.

Reply:
Removed duplicate permutations of tests